web_kanban_stage/README.rst

```diff
@@ -0,0 +1,107 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
Let's license this LGPL. Easy way to change is just replace in directory `AGPL` for `LGPL` and `agpl` for `lgpl`